shared/common-adapters/tab-bar.js.flow

```diff
@@ -47,12 +47,12 @@ export type TabBarButtonProps = Exact<{
   badgeNumber?: ?number,
   badgePosition?: TabBadgePosition,
   style?: Object,
-  styleContainer?: Object,
-  styleBadge?: Object,
-  styleBadgeContainer?: Object,
-  styleIcon?: Object,
-  styleBadgeNumber?: Object,
-  styleLabel?: Object,
+  styleContainer?: any,
```

Comment:
Why not the new style types?

Reply:
we'll slowly be rolling this out. i want to hit the common adapters first since they're the most used so we can catch more errors